database.c

```diff
@@ -712,7 +743,7 @@ void read_data_from_DB(void)
 		queries[queryIndex].clientID = clientID;
 		queries[queryIndex].forwardID = forwardID;
 		queries[queryIndex].timeidx = timeidx;
-		queries[queryIndex].db = true; // Mark this as already present in the database
+		queries[queryIndex].db = dbid;
 		queries[queryIndex].id = queryID;
```

Comment:
Do we now want to revert this to just 0, like how `development` handles it? The database ID is sufficient for the API to paginate on (for queries already in the database, otherwise it needs the dnsmasq ID).